.circleci/config.yml

```diff
@@ -706,8 +706,8 @@ jobs:
     - DTEST_BRANCH: trunk
     - CCM_MAX_HEAP_SIZE: 1024M
     - CCM_HEAP_NEWSIZE: 256M
-    - REPEATED_UTEST_TARGET: testsome
-    - REPEATED_UTEST_CLASS: null
+    - REPEATED_UTEST_TARGET: test
+    - REPEATED_UTEST_CLASS: ViewTest
```

Comment:
Nit: this runs both `org.apache.cassandra.cql3.ViewTest` and `org.apache.cassandra.db.lifecycle.ViewTest`, when we only need the former. I think this can be avoided by using `testsome` and the qualified class name:
```
REPEATED_UTEST_TARGET: testsome
REPEATED_UTEST_CLASS: org.apache.cassandra.cql3.ViewTest
```
There is nothing wrong with running more tests than the strictly needed, I'm just telling in case we want to run this again, or in the other branches :)

Reply:
Yeah I noticed when I checked the results. I hadn't noticed even when ran locally :facepalm: lol